packages/cli/src/playground/src/pages/workflows/workflow/traces.tsx

```diff
@@ -26,14 +26,7 @@ function WorkflowTracesContent() {
     );
   }
 
-  return (
-    <WorkflowTraces
-      traces={traces}
-      isLoading={firstCallLoading}
-      error={error}
-      sidebarChild={<WorkflowInformation workflowId={workflowId!} />}
```

Comment:
Not needed anymore in the traces table where we only show the traces details and not the workflow details